diff --git a/speedup b/speedup.new
index 22685b0..0a513d6 100644
--- a/speedup
+++ b/speedup.new
@@ -12,7 +12,7 @@ my $SET_NOT_EMPTY= join( '|', qw( pcdata cdata comment)); # set the field
 # depending on the version of perl use either qr or ""
 print STDERR "perl version is $]\n";
 
-my $var= '(\$[a-z_]+(?:\[\d\])?|\$t(?:wig)?->root|\$t(?:wig)?->twig_current|\$t(?:wig)?->{\'?twig_root\'?}|\$t(?:wig)?->{\'?twig_current\'?})';
+my $var= '(\$[a-z_]+(?:\[\d\])?|\$t(?:wig)?->root|\$t(?:wig)?->twig_current|\$t(?:wig)?->\{\'?twig_root\'?\}|\$t(?:wig)?->\{\'?twig_current\'?\})';
 
 my $set_to = '(?:undef|\$\w+|\$\w+->\{\w+\}|\$\w+->\w+|\$\w+->\w+\([^)]+\))';
 my $elt    = '\$(?:elt|new_elt|child|cdata|ent|_?parent|twig_current|next_sibling|first_child|prev_sibling|last_child|ref|elt->_parent)';
